gdbgui/src/js/GdbApi.jsx

```diff
@@ -319,6 +319,18 @@ const GdbApi = {
         cmds.push(GdbApi.get_break_list_cmd())
         return cmds
     },
+    get_attach_process_cmds(processid){
+        // tell gdb which arguments to use when calling the binary, before loading the binary
+        let cmds = [
+                `-target-attach ${processid}`
+            ]
+        // add breakpoint if we don't already have one
+        if(store.get('auto_add_breakpoint_to_main')){
```

Comment:
The [gdb docs](https://sourceware.org/gdb/onlinedocs/gdb/Attach.html) say `The first thing GDB does after arranging to debug the specified process is to stop it.`  so I am not sure if we need to add a breakpoint here. What do you think?